test/loopback-dl.test.js

```diff
@@ -661,8 +661,8 @@ describe('DataSource define model', function() {
 
 });
 
-describe('Load models with base', function() {
-  it('should set up base class via base option', function() {
+describe('Models with Base Model', function() {
+  it('have a base class according to the base option', function() {
```

Comment:
This is due to plurality in the describe, `Models`. I'll reword the describe block to singular.